wavelength-plugins/profile.js

```diff
@@ -274,6 +274,41 @@ exports.commands = {
 			);
 		},
 	},
+
+	pokemon: {
+		add: "set",
+		set: function (target, room, user) {
+			let dex = Dex.data.Pokedex.species;
+			if (!target) return this.parse("/pokemonhelp");
+			let pkmn = target.toLowerCase().replace(' ', '');
+			if (!Dex.data.Pokedex[pkmn]) {
+				return this.errorReply('Not a Pokemon. Check your spelling?');
+			}
+			if (Db.pokemon.has(target)) return this.errorReply("This Pokemon is already listed as your favorite Pokemon on your profile.");
```

Comment:
Feel free to just overide the existing one unless it matches.

Reply:
It currently does anyways lol this code is irrelevant tbh

Reply:
Will remove